app/index.html

```diff
@@ -36,38 +36,28 @@
     <link rel="stylesheet" href="styles/main.css">
   </head>
   <body>
-    <header class="app-bar promote-layer">
-      <div class="app-bar-container">
-        <button class="menu"><img src="images/hamburger.svg" alt="Menu"></button>
-        <h1 class="logo">Web <strong>Starter Kit</strong></h1>
-        <section class="app-bar-actions">
-        <!-- Put App Bar Buttons Here -->
-        <!-- e.g <button><i class="icon icon-star"></i></button> -->
-        </section>
-      </div>
-    </header>
+    <script src="lib/angular/angular.min.js"></script>
+    <script src="lib/angular-ui-router/release/angular-ui-router.min.js"></script>
+    <script src="https://cdn.firebase.com/js/client/2.2.7/firebase.js"></script>
+    <script src="https://cdn.firebase.com/libs/angularfire/1.1.2/angularfire.min.js"></script>
+    <script src="main/main.js"></script>
```

Comment:
@gdomorski Put any script tags pertaining to our files at the bottom of the body.
```
<body>
  <!-- App Dependencies -->
  <script src="lib/angular/angular.min.js"></script>
  <script src="lib/angular-ui-router/release/angular-ui-router.min.js"></script>
  <script src="https://cdn.firebase.com/js/client/2.2.7/firebase.js"></script>
  <script src="https://cdn.firebase.com/libs/angularfire/1.1.2/angularfire.min.js"></script>

  <!-- Page Elements go in between here -->





  <!-- Dev Dependencies -->
  <script src="main/main.js></script>
</body>
```